mesheryctl/pkg/utils/helpers.go

```diff
@@ -158,18 +158,55 @@ func SetFileLocation() error {
 
 //PreReqCheck prerequisites check
 func PreReqCheck() error {
+	//Check whether docker daemon is running or not
+	if err := exec.Command("docker", "ps").Run(); err != nil {
+		log.Info("Docker daemon is not running")
+		//No auto installation of docker for windows
+		if runtime.GOOS == "windows" {
+			return errors.Wrap(err, "please start docker daemon. Run `masheryctl system start` once docker is started.")
```

Comment:
Capitalisation should be taken care of. Also it's `mesheryctl`.
Please make the changes in othe areas too.